JdbcGrpcErrorTest.autocommitPDMLExecuteSql test failure

In #182, we started getting the following [failure](https://github.com/googleapis/java-spanner-jdbc/pull/182/checks?check_run_id=870447086) in the CI tests.

```
[ERROR] Failures: 
[ERROR]   JdbcGrpcErrorTest.autocommitPDMLExecuteSql:221 missing expected exception
[INFO] 
[ERROR] Tests run: 335, Failures: 1, Errors: 0, Skipped: 8
```

--- a/pom.xml
+++ b/pom.xml
@@ -71,7 +71,7 @@
       <dependency>
         <groupId>com.google.cloud</groupId>
         <artifactId>libraries-bom</artifactId>
-        <version>8.0.0</version>
+        <version>8.1.0</version>
         <type>pom</type>
         <scope>import</scope>
       </dependency>

--- a/src/test/java/com/google/cloud/spanner/jdbc/JdbcGrpcErrorTest.java
+++ b/src/test/java/com/google/cloud/spanner/jdbc/JdbcGrpcErrorTest.java
@@ -214,7 +214,8 @@ public class JdbcGrpcErrorTest {
 
   @Test
   public void autocommitPDMLExecuteSql() {
-    mockSpanner.setExecuteSqlExecutionTime(SimulatedExecutionTime.ofException(serverException));
+    mockSpanner.setExecuteStreamingSqlExecutionTime(
+        SimulatedExecutionTime.ofException(serverException));
     try (java.sql.Connection connection = createConnection()) {
       connection.createStatement().execute("SET AUTOCOMMIT_DML_MODE='PARTITIONED_NON_ATOMIC'");
       connection.createStatement().executeUpdate(UPDATE_STATEMENT.getSql());

